ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -164,6 +170,46 @@ protected boolean setupStrokePaint(Paint paint, float opacity) {
     return true;
   }
 
+  /*
+   * clculate stops count from array
+   */
+  private static int getStopsCount(float[] value) {
+    int values = (value.length - 5) / 4;
```

Comment:
Can you actually explain what the calculation is?  Looks like the first five values are used and then the rest are assumed to be divisible by 4.